README.md

```diff
@@ -1 +1,31 @@
-# reactjsdenver.com
\ No newline at end of file
+# reactjsdenver.com
+
+Clone the repo:
+
+```
+$ git clone git@github.com:reactjsdenver/reactjsdenver.com.git
+```
+If you do not have babel, webpack, and webpack-dev-server installed globally
```

Comment:
Yes. that makes perfect sense. I will add them all, including the presets to the dependencies.